packages/menu/src/SubMenu.tsx

```diff
@@ -190,6 +315,18 @@ interface SubMenuProps {
   onExited?: ExitHandler<HTMLElement>;
 
   href?: string;
+
+  /**
+   * Determines whether or not the component will be rendered in dark Theme.
+   *
+   * default: `false`
+   */
+  darkMode?: boolean;
```

Comment:
Same concern here. Unless the consumer can toggle DarkMode for individual SubMenus/MenuItems, I don't think we should have this as a prop